services/movie.py

```diff
@@ -6,9 +7,13 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
 ) -> QuerySet:
     queryset = Movie.objects.all()
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+
     if genres_ids:
         queryset = queryset.filter(genres__id__in=genres_ids)
```

Comment:
Consider checking if `genres_ids` is not an empty list before applying the filter. This will prevent unnecessary database queries when the list is empty.